strawberry/auto.py

```diff
@@ -21,3 +25,25 @@ def __repr__(self):
 
 
 auto: Final = Annotated[Any, StrawberryAuto()]
+
+
+def is_auto(type_: Union[StrawberryAnnotation, StrawberryType, type]):
```

Comment:
Can `is_auto` be a `classmethod` (or `staticmethod`) on `StrawberryAuto` instead of an unbound function?

Reply:
In fact, could we just implement `StrawberryAuto.__instancecheck__`?

Reply:
Awesome idea! I went the same road for `UNSET`/`is_unset` here, but maybe we should change it too in the future?

Btw, its done!

Reply:
Yep definitely think we should do that in the future :+1:. Personally, I don't think unbound functions are good practice in Python in anything other than simple scripts.